src/pages/Settings/SettingsEmailPassword/SettingsEmailPassword.css

```diff
@@ -0,0 +1,147 @@
+.container--white {
+    background: #fff;
+}
+
+.settings-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.settings-wrapper--top-pad {
+    padding-top: 15px;
+}
+
+.form {
+    width: 30vw;
+}
+
+.form--settings {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 400;
+}
+
+.form__section {
+    margin: 12px 0;
+    padding-bottom: 5px;
+}
+
+/* .form__label {
+} */
+
+.form__label--settings {
+    display: block;
+    margin: 15px 0 5px 0;
+    padding-top: 20px;
+}
+
+.form__input {
+    box-sizing: border-box;
+    border: none;
+}
+
+.form__input--settings {
+    display: block;
+    width: 100%;
+    margin: 10px 0;
+    padding: 0.5rem;
+    font-size: 1rem;
+    border-bottom: 2px solid grey;
+}
+
+.form__input--settings:hover,
+.form__input--settings:focus,
+.form__input--settings:active {
+    outline: 0;
+    border-bottom: 2px solid #6831de;
+}
+
+.btn {
+    text-align: center;
+}
+
+.btn:hover {
+    cursor: pointer;
+}
+
+.btn--settings {
+    border: transparent;
+    padding: 10px 15px;
+    border-radius: 10px;
+    height: 30px;
+    line-height: 12px;
+    font-size: 1rem;
+    font-weight: bold;
+    box-shadow: 3px 6px 10px #1113;
+}
+
+.btn--settings:hover,
+.btn--settings:focus {
+    opacity: 0.8;
+}
+
+.btn--purple {
+    background-image: linear-gradient(to right, #cf48ff, #6831de);
+    color: #fff;
+}
+
+.btn--disabled {
+    background-color: lightgray;
+    color: gray;
+}
+
+@media only screen and (min-width: 768px) and (max-width: 1024px) {
+    .container--white {
+        width: 100vw;
+        height: 100vh;
+    }
+
+    .settings-wrapper--top-pad {
+        padding-top: 15vh;
+    }
+
+    .settings-wrapper {
+        padding-right: 250px;
```

Comment:
padding-right: 250px, Does not look good on tablet rotated. We are not too concerned about how it looks on the landspace view. What does this padding-right: 250px was meant to accomplish?
![chrome-tools-pd250](https://user-images.githubusercontent.com/38657595/58865306-06a71700-86bf-11e9-969b-d373f1540868.jpg)